v2/thirdparty/custom-ui/thirdparty-login.mdx

```diff
@@ -764,75 +852,75 @@ On success, the backend will send back session tokens as part of the response he
 
```

Comment:
the pkce link in RN is broken

Reply:
Also, remove the ref to google in that section, and make the section like "Some providers prefer that you use PKCE, in which case you can use .. lib and set .. to true etc etc.

Reply:
The important thing under it should say that you need to pass in the redirect uri that matches what you have configured on the dashboard.

Reply:
Also, change other mobile code snippets to not mention google in this section + link to generic oauth client lib